ms_gsl.sh

```diff
@@ -0,0 +1,31 @@
+package: ms_gsl
+version: master
+source: https://github.com/Microsoft/GSL.git
+tag: master
+---
+#!/bin/bash -e
+
+# recipe for the C++ guidelines support library (Microsoft implementation)
+# can be deleted once we are fully C++17 compliant
+
+# just rsync into the installdir
+rsync -a $SOURCEDIR/ $INSTALLROOT/
```

Comment:
I think you need to exclude `.git`, no? Also mention the fact that this is header only.

Reply:
fixed